src/frameName.cpp

```diff
@@ -135,13 +136,16 @@ char* FrameName::truncate(char* name, int max_length) {
 
 const char* FrameName::decodeNativeSymbol(const char* name) {
     const char* lib_name = (_style & STYLE_LIB_NAMES) ? Profiler::instance()->getLibraryName(name) : NULL;
+    bool report_lib_name_from_symbol = lib_name == NULL && strcmp(name, "call_stub") != 0 &&
+        !StringUtils::endsWith(name, "_[k]", 4) &&
+        Profiler::instance()->findLibraryByName(name) != NULL;
```

Comment:
To my understanding, `findLibraryByName` gets a `CodeCache` by lib basename, and `getLibraryName` gets a lib basename from a native function name.  So how come you're calling both functions with the same variable?

Reply:
I'm checking for the case that variable `name` stores only the path (or "name" of the library).
But I think I can introduce a slight change to reduce the confusion.

Reply:
Yes, better now